docs/synapse/userguides/storm_adv_methods.rstorm

```diff
@@ -203,6 +204,50 @@ The method takes no arguments.
 
 .. storm-cli:: inet:dns:a=(woot.com,54.173.9.236) $lib.print($node.ndef()) | spin
 
+.. _meth-node-path:
+
+$node.path
```

Comment:
So for `$path`, we renamed the type of the object to `node:path` but that didn't actually change the way it was accessed ($node.path doesn't exist). Not sure if this was something that was discussed and we want to implement (it does seem like a good place for it) but currently we've only changed the type name which was what broke the docs references.

Reply:
Well that was not at all how I understood  it :rofl: